test/e2e/standard/cluster.go

```diff
@@ -333,6 +333,7 @@ func (sc *SanityChecker) checkCanAccessService(ctx context.Context, url string,
 	}
 
 	if sc.cs.Properties.NetworkProfile.PrivateEndpoint != nil && usePE {
+		inner.TLSClientConfig.InsecureSkipVerify = true
```

Comment:
Please don't do this even in test - override DialTLS - copy roundtrippers.HealthCheck